lib/collection/request.js

```diff
@@ -83,6 +83,11 @@ _.assign(Request.prototype, /** @lends Request.prototype */ {
         _.has(options, 'method') && (this.method = _.isString(options.method) ? options.method.toUpperCase() :
             (options.method || DEFAULT_REQ_METHOD));
 
+        if (_.has(options, 'body')) {
+            _.isString(options.body) && (options.body = { mode: 'raw', raw: options.body });
```

Comment:
Not sure about the self-correction here.

Reply:
As discussed offline.